packages/web/lib/search/search-filters.ts

```
@@ -0,0 +1,368 @@
+          } else if (value.toLowerCase() === 'note') {
+            filters.isFolder = false
+          }
+          break
+        case 'has':
+          if (value.toLowerCase() === 'content') {
+            filters.hasContent = true
+          }
+          break
+        case 'date':
+          // Handle special date values
+          const datePresets = new SearchFilters().getDateRangePresets()
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)